spec/javascript-spec.coffee

```diff
@@ -85,6 +85,13 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('0O1411')
       expect(tokens[0]).toEqual value: '0O1411', scopes: ['source.js', 'constant.numeric.js']
 
+    it "regular expressions have explicit count modifiers", ->
```

Comment:
Specs should flow with the `it`, eg `it "tokenizes regular expressions that have explicit count modifiers"`
This obviously doesn't work here since it's not testing tokenization but it's an example.